### PR TITLE
feat: style the explorer site

### PR DESCRIPTION
**Pull request by eljefe223**
# Description
Styling updates to match designs.

Screen Shot 2019-07-26 at 9 39 58 AM

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [x] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/fast-component-explorer/app/explorer.style.ts [hunk 1]
```diff
@@ -31,22 +40,59 @@ const style: ComponentStyles<ExplorerClassNameContract, DesignSystem> = {
...
+            background: neutralFillActive(neutralLayerL2),
         },
         "&:focus": {
             outline: "none",
         },
     },
-    explorer_viewerRegion: {
+    explorer_devToolsPanel: {
+        boxShadow:
+            "0px 0.6px 1.8px rgba(0, 0, 0, 0.11) inset, 0px 3.2px 7.2px rgba(0, 0, 0, 0.13) inset",
```
Comment (nicholasrice) [inline packages/fast-component-explorer/app/explorer.style.ts]: Are these custom, or are these related to elevation?

Comment (eljefe223) [reply packages/fast-component-explorer/app/explorer.style.ts]: Custom, the shadows need to come from specific directions.

File: packages/fast-component-explorer/app/explorer.style.ts [hunk 2]
```diff
@@ -31,22 +40,59 @@ const style: ComponentStyles<ExplorerClassNameContract, DesignSystem> = {
...
+        height: "22px",
         ...applyCornerRadius(),
         "&:hover": {
-            background: neutralFillHover(() => "#3B3B3B"),
+            background: neutralFillHover(neutralLayerL2),
         },
         "&:active": {
-            background: neutralFillActive(() => "#3B3B3B"),
+            background: neutralFillActive(neutralLayerL2),
         },
```
Comment (chrisdholt) [inline packages/fast-component-explorer/app/explorer.style.ts]: nit / fyi, we can likely chain these selectors:

```
“&:hover, &:active”: {}
```

Comment (eljefe223) [reply packages/fast-component-explorer/app/explorer.style.ts]: the hover and active colors are different

File: packages/fast-component-explorer/app/explorer.style.ts [hunk 3]
```diff
@@ -31,22 +40,59 @@ const style: ComponentStyles<ExplorerClassNameContract, DesignSystem> = {
...
+        boxShadow:
+            "-0.6px 0px 1.8px rgba(0, 0, 0, 0.11) inset, -3.2px 0px 7.2px rgba(0, 0, 0, 0.13) inset",
+    },
+    explorer_paneTitleContainer: {
+        height: "32px",
         display: "flex",
+        boxSizing: "border-box",
+        alignItems: "center",
+        paddingLeft: "10px",
+        borderBottom: format<DesignSystem>("1px solid {0}", neutralDividerRest),
```
Comment (chrisdholt) [inline packages/fast-component-explorer/app/explorer.style.ts]: instead of 1px here, should we map this to the design system value for outlineWidth?

File: packages/fast-component-explorer/app/explorer.style.ts [hunk 4]
```diff
@@ -31,22 +40,59 @@ const style: ComponentStyles<ExplorerClassNameContract, DesignSystem> = {
...
+    explorer_propertiesPanel: {
+        background: neutralLayerL3,
+        boxShadow:
+            "0.6px 0px 1.8px rgba(0, 0, 0, 0.11) inset, 3.2px 0px 7.2px rgba(0, 0, 0, 0.13) inset",
+    },
+    explorer_toolbar: {
+        padding: "0 10px",
+        height: "32px",
+        display: "flex",
+        verticalAlign: "middle",
```
Comment (chrisdholt) [inline packages/fast-component-explorer/app/explorer.style.ts]: Is vertical align middle doing anything here? With the flex alignment to center everything should be aligned as expected without verticalAlign (typically used w/ inline layouts)

File: packages/fast-component-explorer/app/explorer.tsx [hunk 5]
```diff
@@ -176,84 +186,106 @@ class Explorer extends Foundation<ExplorerHandledProps, {}, ExplorerState> {
...
-        );
         return (
-            <DesignSystemProvider designSystem={explorerDesignSystem}>
-                <Background value={this.backgrounds.L1}>
-                    <Container className={get(this.props, "managedClasses.explorer")}>
-                        <Row style={{ flex: "1" }}>
-                            <Pane>
+            <Background value={neutralLayerL1}>
+                <Container className={get(this.props, "managedClasses.explorer")}>
+                    <Row style={{ flex: "1" }}>
```
Comment (chrisdholt) [inline packages/fast-component-explorer/app/explorer.tsx]: any specific reason for calling this inline instead of using a className as elsewhere?

File: packages/fast-component-explorer/app/explorer.style.ts (hunk 2 above)
Comment (chrisdholt) [reply packages/fast-component-explorer/app/explorer.style.ts]: Ah - doh! I see the recipe is different. My mind went blank at neutralFill

File: packages/fast-component-explorer/app/explorer.tsx (hunk 5 above)
Comment (eljefe223) [reply packages/fast-component-explorer/app/explorer.tsx]: It was already there and hardcoded. If we need to add more styling to it later sure.

File: packages/fast-component-explorer/app/explorer.style.ts (hunk 3 above)
Comment (eljefe223) [reply packages/fast-component-explorer/app/explorer.style.ts]: None of this stuff needs to map to the designSystem only the colors,

File: packages/fast-component-explorer/app/explorer.style.ts (hunk 4 above)
Comment (eljefe223) [reply packages/fast-component-explorer/app/explorer.style.ts]: yeah I'll remove
